.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - run:
           name: Build elabtmp image and start containers
           command: |
-            docker build -q --build-arg BASE_IMAGE_VERSION=${CIRCLE_BRANCH} -t elabtmp -f tests/elabtmp.Dockerfile .
+            docker build -q --no-cache -t elabtmp -f tests/elabtmp.Dockerfile .
```

Comment:
is this a git merge issue?

Reply:
Sorry for the confusion. Your approach to use CIRCLE_BRANCH to get the image from docker only works for master, next, and hypernext but fails for all other branch names. So basically for every PR.
I realized it because the tests failed on circle. On scrutinizer there is another problem either because of the changes to the config file or because they upgraded the default to php8.2.

For circle I removed the env var because I wanted to see if it will pass at all.
Maybe we can pull hypernext from docker by default and only change to next or master if the name matches. But what if a new image is not uploaded yet?

Reply:
yeah I see now how broken it is. In fact this was only made so the `noconfig` branch would correctly be tested with the `noconfig` branch of elabimg, because hypernext was incompatible. But it is quite rare that they are incompatible so maybe we can just forget about this and go with hypernext all the time, it shouldn't be a big issue.